tests/adapters/switches/dell_test.py

```diff
@@ -235,6 +243,13 @@ def test_get_vlans_multipage(self):
 
         assert_that(vlans, has_length(10))
 
+        assert_that(vlans[0].interfaces, equal_to(['port-channel 2', 'port-channel 3', 'port-channel 5', 'port-channel 6', 'port-channel 8', 'port-channel 9',
```

Comment:
Franky ch8-10 is well enough, so you would be testing ranges of 1, 2 and 3... any number more than 3 is irrelevant and crowds the test